common/lib/capa/capa/templates/choicetext.html

```diff
@@ -59,10 +59,12 @@
         <span id="answer_${id}"></span>
     </fieldset>
         <input class= "choicetextvalue" type="hidden"  name="input_${id}{}" id="input_${id}" value="${value|h}" />
-    
+
     <div class="indicator-container">
     % if input_type == 'checkbox' or not element_checked:
-        <span class="status ${status.classname}" id="status_${id}"></span>
+        <span class="status ${status.classname}" id="status_${id}">
+            <span class="sr">${_(status.classname)}</span>
```

Comment:
@alisan617 

The problem types failing all connect this this file, choicetext.html. By changing the line above to something like 

`${status.display_name}` 

You will get the translated display name of the status from the inputtypes.py. 

`<span class="sr">unanswered</span>`

Once this is corrected the bok_choy and python tests should start passing.